templates/document-definitions-shell-template.js

```diff
@@ -1,4 +1,5 @@
 function(require) {
+  var _ = require('../lib/underscore/underscore-min.js');
```

Comment:
Makes Underscore.js available to document definitions that are to be inspected by the work-in-progress document definitions validation script.